.github/workflows/ci.yml

```diff
@@ -312,6 +313,16 @@ jobs:
           path: ./logs-kind
           if-no-files-found: error
           retention-days: 7
+      
+      - name: Publish E2E Test Flakes
+        if: '!cancelled()'
+        uses: buildpulse/buildpulse-action@v0.11.0
+        with:
+          account: 45158470
+          repository: 147886080
+          key: ${{ secrets.BUILDPULSE_ACCESS_KEY_ID }}
```

Comment:
looking at other people's [pipelines](https://github.com/Kong/kubernetes-ingress-controller/blob/60a60159cf289b0c6d54b65ea0f86670659f4448/.github/workflows/_test_reports.yaml#L68) it looks like it should work. But it's strange that in their case it shows them as set:
![image](https://github.com/crossplane/crossplane/assets/5697904/56a44738-f5d4-4248-9ae8-ba51da1abc96)
While here it doesn't:
![image](https://github.com/crossplane/crossplane/assets/5697904/c7256706-d01a-480a-aef6-ba525e90c24b)

Reply:
I wonder if it's not some gha shenanigan and it'll work once it's on main 🤔

Reply:
aha! `Anyone with collaborator access to this repository can use these secrets and variables for actions. They are not passed to workflows that are triggered by a pull request from a fork.`

Reply:
Unfortunately it looks like it'll only work when they implement tokenless upload as codecov did [here](https://github.com/codecov/codecov-action/issues/29). But given that it's actually ignored, as even if it's failing, that step is marked as successful, we can leave it as is, maybe one day it'll be fixed upstream and will work also for PRs, or maybe it's already working fine, but I don't know where to look at 😅

Reply:
Thanks for investigating @phisco! so since CI is run on `master` after each PR is merged, we'll probably start getting data on history master and that will be a good place to start 💪